Source/swiftlint/LintCommand.swift

```diff
@@ -21,36 +21,43 @@ struct LintCommand: CommandType {
 
     func run(mode: CommandMode) -> Result<(), CommandantError<()>> {
         return LintOptions.evaluate(mode).flatMap { options in
+            let configuration = Configuration(path: options.configurationFile,
+                optional: !Process.arguments.contains("--config"))
             if options.useSTDIN {
                 let standardInput = NSFileHandle.fileHandleWithStandardInput()
                 let stdinData = standardInput.readDataToEndOfFile()
                 let stdinNSString = NSString(data: stdinData, encoding: NSUTF8StringEncoding)
                 if let stdinString = stdinNSString as? String {
-                    let violations = Linter(file: File(contents: stdinString)).styleViolations
-                    print("\n".join(violations.map { $0.description }))
+                    let file = File(contents: stdinString)
+                    let linter = Linter(file: file, configuration: configuration)
+                    print("\n".join(linter.styleViolations.map { $0.description }))
```

Comment:
The `swift-2.0` branch is targeting Xcode 7 beta 5. I'll have to update all the dependencies to beta 6 before I can do the same to SwiftLint... ugh.